doc/source/_static/css/custom.css

```diff
@@ -52,3 +52,8 @@ img.inline-figure {
   /* Make version warning clickable */
   z-index: 1;
 }
+
+/* Adjustment to Version block */
+.rst-versions {
+  z-index: 1200;
```

Comment:
I think `z-index: 1030;` will fix this

Reply:
how did you figure that out?

Reply:
haha I couldn't find the z-index of the table of contents so I kind of just binary-searched with trial/error to find the lowest possible z-index

Reply:
I will admit that `1200` instead of `1030` is not likely to cause any problems in the future

Reply:
haha will keep as is..